app/views/users/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+  <aside>
+    <section class="user_info">
+      <h1>Name: <%= @user.name %></h1>
+      <h3>Username: <%= @user.username %></h3>
+    </section>
+  </aside>
+
+  <div>
+   <h4 class="index-heading">Created Events</h4>
+   <%= render 'user_events', events: @user.created_events %>
+  </div>
+
+  <div>
+   <h4 class="index-heading">Attending Events</h4>
+   <%= render 'user_events', events: @user.attending_events %>
+  </div>
+</div>
```

Comment:
- [ ] There should be NO application logic in the views nor helpers (no `Post.new`, `Comment.create`, `current_user.friends.find_by(...)`...). Everything connected with data processing should be encapsulated in models and used in controllers.

_These are making database queries and show not be in the view._

The show action of your user controller should have things like:

* [ ] lists events created by user, i.e. `@created_events = current_user.events` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_show_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
* [ ] lists events previoulsy attendented by user **using event model scope**, i.e. `@past_events = current_user.attended_events.previous` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
* [ ] lists upcoming events previously attended by user **using event model scope**, i.e. `@upcoming_events = current_user.attended_events.upcoming` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))